packages/mgt-components/src/components/mgt-agenda/mgt-agenda.scss

```diff
@@ -9,198 +9,265 @@
 @import '../../styles/shared-styles.scss';
 @import './mgt-agenda.theme.scss';
 
-:host .header,
-mgt-agenda .header {
-  margin: var(--agenda-header-margin, 40px 10px 14px 10px);
-  font-family: $font-family;
-  font-size: var(--agenda-header-font-size, 24px);
-  color: $header__color;
-}
+:host,
+mgt-agenda {
+  --card-height: auto;
 
-:host .agenda > .group:first-child > .header,
-mgt-agenda .agenda > .group:first-child > .header {
-  margin-top: 0;
-}
+  .header {
+    margin: var(--agenda-header-margin, 18px 0px 12px 10px);
+    font-family: $font-family;
+    font-size: var(--agenda-header-font-size, 24px);
+    font-style: normal;
+    font-weight: 400;
+    line-height: 32px;
+    color: $header__color;
+    opacity: 0.9;
+  }
 
-:host .agenda-list,
-mgt-agenda .agenda-list {
-  list-style-type: none;
-  padding: 0;
-  margin: 0;
-  font-family: $font-family;
-  font-style: normal;
-  font-weight: normal;
-}
+  .agenda,
+  .group {
+    display: flex;
+    flex-direction: column;
+    row-gap: var(--event-row-gap, 14px);
 
-:host .event,
-mgt-agenda .event {
-  background: set-var(event__background-color, $theme-default, $agenda);
-  border: var(--event-border, solid 2px rgba(0, 0, 0, 0));
-  box-shadow: var(--event-box-shadow, 0px 2px 8px set-var(box-shadow__color, $theme-default, $common));
-  margin: var(--event-margin, 0px 10px 14px 10px);
-  padding: var(--event-padding, 0px);
-  display: flex;
-  border-radius: 8px;
-}
+    & > .group:first-child > .header {
+      margin-top: 0;
+    }
 
-:host .narrow .event,
-mgt-agenda .narrow .event {
-  flex-direction: column;
-}
+    .event {
+      background: set-var(event__background-color, $theme-default, $agenda);
+      border: var(--event-border, solid 2px rgba(0, 0, 0, 0));
+      box-shadow: var(--elevation-shadow-card-rest, --event-box-shadow);
+      padding: var(--event-padding, 12px);
+      position: relative;
+      display: flex;
+      flex: 1 1 auto;
```

Comment:
```suggestion
      flex: 1 1 auto;
      content-visibility: visible;
      contain: none;
```

Reply:
This seems to work based on my editing CSS in the browser tools.

The one immediate drawback I see is that it introduces a horizontal scrollbar in storybook, so it might be necessary to investigate why that is and if we can eliminate the horizontal scrollbar.